utilities/src/main/java/io/syndesis/qe/templates/MysqlTemplate.java

```diff
@@ -0,0 +1,134 @@
+package io.syndesis.qe.templates;
+
+import io.fabric8.kubernetes.api.model.ContainerPort;
+import io.fabric8.kubernetes.api.model.ContainerPortBuilder;
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.fabric8.kubernetes.api.model.IntOrString;
+import io.fabric8.kubernetes.api.model.ServicePortBuilder;
+import io.fabric8.kubernetes.api.model.ServiceSpecBuilder;
+import io.syndesis.qe.accounts.Account;
+import io.syndesis.qe.accounts.AccountsDirectory;
+import io.syndesis.qe.utils.OpenShiftUtils;
+import io.syndesis.qe.wait.OpenShiftWaitUtils;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeoutException;
+
+@Slf4j
+public class MysqlTemplate {
+
+
+    private static final String APP_NAME = "mysql";
+    private static final String LABEL_NAME = "app";
+    private static final String DB_USER = "developer";
+    private static final String DB_PASSWORD = "developer";
+    private static final String DB_SCHEMA = "sampledb";
+    private static final String DB_URL = "jdbc:mysql://mysql:3306/sampledb";
+
+    public static void deploy() {
+
+        List<ContainerPort> ports = new LinkedList<>();
+        ports.add(new ContainerPortBuilder()
+                .withName("mysql-cmd")
+                .withContainerPort(3306)
+                .withProtocol("TCP").build());
+
+        Map<String, String> templateParams = new HashMap<>();
+        templateParams.put("MYSQL_USER", DB_USER);
+        templateParams.put("MYSQL_PASSWORD", DB_PASSWORD);
+        templateParams.put("MYSQL_DATABASE", DB_SCHEMA);
+
+        Collection<EnvVar> params = MysqlTemplate.repackAllParams(templateParams);
```

Comment:
Why do we actually need templateParams in map first and then remap them into EnvVar list. Creating a list of EnvVar is not and option here?